src/styles/_header.scss

```
@@ -0,0 +1,64 @@
+@import "variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+  height: 60px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-icon {
+  padding: 10px 0 10px 50px;
+}
+
+.nav {
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+
+  white-space: nowrap;
+}
+
+.nav__element {
+  list-style-type: none;
+}
+
+.nav__list > .nav__element:last-child > a {
```

Задавай марджини `li`  і тоді непотрібно писати складні селектори
```suggestion
.nav__list > .nav__element:last-child > a {
```

---

Спростив селектор.